src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/user/AcknowledgeClaimCallbackHandler.java

```diff
@@ -38,8 +39,10 @@
     private static final List<CaseEvent> EVENTS = Collections.singletonList(ACKNOWLEDGE_CLAIM);
 
     public static final String CONFIRMATION_SUMMARY = "<br />You need to respond to the claim before %s."
-        + "\n\n[Download the Acknowledgement of Claim form](%s)";
+        + "%n%n[Download the Acknowledgement of Claim form](%s)"
+        + "%n%n<br/><br/>This is a new service - your <a href=\"%s\" target=\"_blank\">feedback</a> will help us to improve it.";
```

Comment:
@Service
class ExitSurveyContentGenerator {
    
 private final ExitSurveyConfiguration exitSurveyConfiguration;

public String applicantSurvey() {
return String.format("%n%n<br/><br/>This is a new service - your <a href=\"%s\" target=\"_blank\">feedback</a> will help us to improve it.", exitSurveyConfiguration.getRespondentSurvey());
}
public String respondentSurvey() {
....
}
}